bin/node-template/node/src/service.rs

```diff
@@ -29,66 +29,52 @@ macro_rules! new_full_start {
 	($config:expr) => {{
```

Comment:
Why do we still have this macro?

Reply:
It exists because it's used both in `new_full!` and `command::run`. I could turn it into a function but the return type would be very complex.